### ISSUE TITLE
Increase time for async operations?

### ISSUE THREAD

**Issue by normancarcamo** [bencompton/jest-cucumber#83]
Hi there, I would love to know if you can tell me where can I set more time on an async operation...

Is there any option like in this example:

```ts
test('User gets OK after file is uploaded', ({ given, when, then, and }) => {
  givenADatasetFile(given);
  andProviderNameIs(and);
  andColumnsAre(and);
  when('the file is uploaded', () => {
    return new Promise((resolve) => {
      req = request.post('/v1/upload');
      req.set('Content-Type', 'application/octet-stream');
      req.set('Accept', 'application/json');
      req.set('Transfer-Encoding', 'chunked');
      req.set('x-csv-provider', provider);
      req.set('x-csv-columns', columns);
      req.on('response', (response) => { res = response; });
      readable.pipe(req as any, { end: false });
      readable.on('end', () => req.end(resolve));
    });
  });
  thenServerRespondsWithStatus(then);
});
```

The error I got is this:
<img width="717" alt="Screen Shot 2020-03-08 at 5 33 25 AM" src="https://user-images.githubusercontent.com/2402579/76161997-6626cd00-60fe-11ea-92cd-8d048847bc0a.png">

```sh
npx envinfo --preset jest

System:
  OS: macOS Mojave 10.14.6
  CPU: (8) x64 Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz
Binaries:
  Node: 12.14.1 - ~/.nvm/versions/node/v12.14.1/bin/node
  npm: 6.13.4 - ~/.nvm/versions/node/v12.14.1/bin/npm
npmPackages:
  jest: ^25.1.0 => 25.1.0
```

Any hint to overcome this?

Thanks.

**Comment by normancarcamo** [comment]
Waiting the approval of the [PR](https://github.com/bencompton/jest-cucumber/pull/85).

**Comment by SirSkorpan** [comment]
Timeouts are controlled by Jest, you should be able to set `testTimeout: <some-numer-of-ms>` in your Jest config (e.g. `jest.config.json`). See more about configuring Jest at [jestjs.io](https://jestjs.io/docs/en/configuration.html#testtimeout-number)

**Comment by normancarcamo** [comment]
@SirSkorpan I really haven't seen that option in that file, but I will, thank you.

**Comment by normancarcamo** [comment]
@SirSkorpan what about the projects that aren't using TS? I think the PR can help in that case or do you suggest anything else? maybe I am missing something.

**Comment by SirSkorpan** [comment]
It shouldn't be a difference between TS and Javascript. As for the PR, I'll let the maintainer decide if it's something that would be interesting to jest-cucumber.

**Comment by bencompton** [comment]
Addressed in #85
